client/app/lib/time.ts

```diff
@@ -14,4 +15,8 @@ function pad2(digit: number): string {
   return digit < 10 ? `0${digit}` : `${digit}`
 }
 
-export { timeFromString, timeToString }
\ No newline at end of file
+const isStartTimeBeforeEndTime = (startTime: string, endTime: string) => {
```

Comment:
```suggestion
const isBefore = (startTime: string, endTime: string) => {
```

The parameter names makes it descriptive already; so we can shorten it

Reply:
no problem. will update

Reply:
For consistency please use `function` keyword for functions and reserve arrow functions for lambdas aka inline callback functions.

@avi-l